src/helpers/district-designer.js

```diff
@@ -17,10 +17,17 @@ export const startAlgorithm = (algoType, shortName, weights) => {
       return JSON.parse(request.response);
     }
   }
-
+  console.log(body);
   request.open("POST", URL + "/StartAlgorithm", false);
-  request.send(body);
-  return request.onreadystatechange();
+  try{
+    request.send(body);
+  } catch (e) {
+    if (e.name === 'NetworkError') {
+      console.log('Network Error');
+      return false;
+    }
+  }
+  return true;//request.onreadystatechange();
```

Comment:
I know it's bad form to push commented code. This is here because this was a dirty fix to make it work before the code review, and should be changed later

Reply:
I'm going to take another look at this before class tomorrow to clean it up